webpack/overrides.js

```diff
@@ -1,11 +1,16 @@
 const { merge, remove } = require('lodash')
+const webpack = require('webpack')
 
 const helpers = require('../helpers')
 const customConfig = require('./webpack.config')
 
 module.exports = (defaultConfig) => {
   let config = merge({}, defaultConfig, customConfig)
 
+  config.plugins.unshift(new webpack.DefinePlugin({
+    __DEV__: JSON.stringify(process.env.NODE_ENV !== "production"),
```

Comment:
Why do you need to stringify a Boolean ?

Reply:
Because I [copy-pasted](https://github.com/apollographql/react-apollo-error-template/pull/51) it :sweat_smile:

Reply:
Hint: I never use double quotes.

Reply:
But this results in a string instead of boolean 🤔 Weird...